frontend/src/scenes/onboarding/OnboardingProductIntroduction.tsx

```diff
@@ -0,0 +1,250 @@
+import './onboarding.scss'
+
+import { IconCheck, IconMap, IconMessage, IconStack } from '@posthog/icons'
+import { LemonButton, Link, Spinner } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { WavingHog } from 'lib/components/hedgehogs'
+import React from 'react'
+import { convertLargeNumberToWords } from 'scenes/billing/billing-utils'
+import { billingProductLogic } from 'scenes/billing/billingProductLogic'
+import { ProductPricingModal } from 'scenes/billing/ProductPricingModal'
+import { getProductIcon } from 'scenes/products/Products'
+import { SceneExport } from 'scenes/sceneTypes'
+import { urls } from 'scenes/urls'
+
+import { BillingProductV2Type, BillingV2FeatureType, ProductKey } from '~/types'
+
+import { onboardingLogic } from './onboardingLogic'
+
+export const scene: SceneExport = {
+    component: OnboardingProductIntroduction,
+    logic: onboardingLogic,
+}
+
+export const Feature = ({ name, description, images }: BillingV2FeatureType): JSX.Element => {
+    return images ? (
+        <li className="text-center">
+            <div className="mb-2 w-full rounded">
```

Comment:
Might look nice to add a subtle box-shadow around each element to help it stand out against the background

I copied the first box-shadow vs code search returned 😅 : `box-shadow: 0 0 30px rgb(0 0 0 / 20%);`

<img width="818" alt="Screenshot 2024-01-25 at 12 53 08 PM" src="https://github.com/PostHog/posthog/assets/21014901/a0f4af82-4302-4cfa-becd-16c76f0010a2">

Reply:
@corywatilo?

I don't think we have many shadows anywhere else.